limesurvey/limesurvey.py

```diff
@@ -132,6 +133,25 @@ def get_survey(self, data, suffix=""): # pylint: disable=unused-argument
             "access_code": self.access_code,
         }
 
+    def user_in_survey(self, anonymous_user_id: str) -> bool:
+        """
+        Check if the user is already in the survey.
+
+        Args:
+            anonymous_user_id (str): The anonymous user ID
+        """
+        params = [
+            self.survey_id, 0, 1, False,
+            ["attribute_1"], {"attribute_1": anonymous_user_id}
+        ]
```

Comment:
```suggestion
        params = [
            self.survey_id, 0, 1, False, ["attribute_1"], {"attribute_1": anonymous_user_id},
        ]
```

Reply:
what are those params? they look a bit random. Can we document them in the method docstring?

Reply:
Done. The `params` variable was documented